docs/software/devtools/vihps.md

```diff
@@ -0,0 +1,138 @@
+[](){#ref-devtools-vihps}
+# VI-HPS tools
+
+The [VI-HPS](https://www.vi-hps.org/tools) Institute (Virtual Institute for High Productivity Supercomputing) provides tools that can assist developers of simulation codes to address their needs in performance analysis.
+
+## [Score-P](https://www.vi-hps.org/projects/score-p/overview/overview.html)
+
+![logo_scorep](../../images/devtools/vihps/logo_scorep.png){ width="5%"}
+is a highly scalable instrumentation and measurement infrastructure for profiling, event tracing, and online analysis. It supports a wide range of HPC platforms and programming models. Score-P provides core measurement services for a range of specialized analysis tools, such as Vampir, Scalasca and others.
+
+## [Vampir](https://www.vi-hps.org/tools/vampir.html) 
+
+![logo_vampir](../../images/devtools/vihps/logo_vampir.png){ width="5%"}
+is a performance visualizer that allows to quickly study the program runtime behavior at a fine level of details. This includes the display of detailed performance event recordings over time in timelines and aggregated profiles. Interactive navigation and zooming are the key features of the tool, which help to quickly identify inefficient or faulty parts of a program.
+
+!!! info
+    While Score-P does not require a license, [Vampir](https://vampir.eu/licensing) does. CSCS standard license allows to read trace files with up to 256 concurrent threads of execution.
+
+    The Vampir GUI is currently available only on `x86-64` CPU based systems and is not provided via a uenv (more details in the Quickstart guide below).
+    You can use Score-P to generate OTF2 traces files on Alps compute nodes and then visualize the results with Vampir on a x86-64 CPU based system (for instance Eiger, LUMI or your own license). 
+
+## [Cube and Scalasca](http://www.vi-hps.org/tools/scalasca.html)
+
+![logo_scalasca](../../images/devtools/vihps/logo_scalasca.png){ width="5%"}
+ support the performance optimization of parallel programs with a collection of scalable trace-based tools for in-depth analyses of concurrent behavior. The analysis identifies potential performance bottlenecks - in particular those concerning communication and synchronization - and offers guidance in exploring their causes.
+
+## Quickstart guide
+
+The VI-HPS uenv is named `scorep` and it can be loaded into your environment as explained here and in the [uenv documentation][ref-uenv].
+
+??? example "Finding and pulling available `scorep` versions"
```

Comment:
How about expanding these boxes? All of the information in them is neccesary/good for readers to see without having to click